admin/lib/validator/pins-overview-validator.js

```diff
@@ -0,0 +1,22 @@
+const serviceManagerErrorMessages = require('../errors/service-manager')
+const testDeveloperErrorMessages = require('../errors/test-developer')
+
+/**
+ * Validate check window and related check forms
+ * @param {Object} checkWindow
+ * @param {Array} allForms
+ * @returns {String}
+ */
+module.exports.validate = (checkWindow, allForms) => {
```

Comment:
This doesn't follow the front end validator pattern we have.  Validators should return a ValidatorError object that is either empty or contains errors.

Also - this appears to be named incorrectly - it is validating that there is a checkWindow (but not that it is in date) and the check Forms.  So we might call it something like CheckWindowAndCheckFormValidator.   But if we do that, we see that it might actually be better to create 2 validators - one for checkWindow and the other for CheckForms.

However, the current implementation of ValidationError is more geared towards showing errors from user-input in forms, which is what the other validators do, whereas what's needed here is to validate other entities in the DB - a set of predicates.

Is this even a validator then? It might be better to move this all to the service and call the service something like `checkWindowSanityCheckService.check()` that returns the error string.  @GuyHarwood - thoughts?    If we need to validate check windows here it seems likely we need to sanity check them elsewhere too.